setup.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+from setuptools import setup
+
+setup(
+    name='cobalt-transcribe',
+    python_requires='>=3.5.0',
+    version='1.0.0',
+    description="This client library is designed to support Cobalt's speech recognition model 'Cobalt Transcribe'.",
+    author='Cobalt Speech and Language Inc.',
+    maintainer_email='tech@cobaltspeech.com',
+    url='https://docs-v2.cobaltspeech.com/docs/asr/transcribe/',
+    packages=["cobaltspeech/transcribe/v5"],
```

Comment:
You would need to import `find_packages` for this : `from setuptools import setup, find_packages`

```suggestion
    packages=find_packages(include=["cobaltspeech", "cobaltspeech.*"]),
```